modules/spring-config/src/main/java/org/mule/runtime/config/internal/dsl/declaration/DefaultXmlArtifactDeclarationLoader.java

```diff
@@ -728,7 +730,26 @@ private void handleInfrastructure(final ParameterModel paramModel,
         findAnyMatchingChildById(declaredConfigs, TLS_CONTEXT_ELEMENT_IDENTIFIER)
             .ifPresent(config -> {
               ParameterObjectValue.Builder tls = newObjectValue();
-              cloneAsDeclaration(config, tls);
+
+              copyExplicitAttributes(config.getConfigAttributes(), tls);
+
+              config.getChildren().forEach(child -> {
+                ParameterObjectValue.Builder childBuilder = newObjectValue();
+
+
+                if (child.getIdentifier().equals(TLS_REVOCATION_CHECK_ELEMENT_IDENTIFIER)) {
+
+                  cloneAsDeclaration(child.getChildren().get(0),
```

Comment:
partial apps that are deployed for DS may not have this child element, you have to check if an element exists before accessing it. If none is present just continue, no failure.